src/main/java/com/fakng/fakngagrgtr/parser/apple/LocationApple.java

```diff
@@ -0,0 +1,35 @@
+package com.fakng.fakngagrgtr.parser.apple;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@Data
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class LocationApple {
+    @JsonProperty("city")
```

Comment:
You don't need to define these annotations, if you remove the one above your class. @JsonNaming overrides the default strategy for reading JSON. By default it was camel case, but you copied this one from another class and now it expects snake_case
That's why you needed to define all those annotations on every field